application/frontend/components/BaseRestController.php

```diff
@@ -36,7 +36,7 @@ public function actionIndex()
         $this->throwNotAllowed();
     }
 
-    public function actionUpdate()
+    public function actionUpdate($id)
```

Comment:
In Yii1, it was typically just a standard controller/action automatic mapping, something like the following (with a few other custom routes):
```
'<controller:\w+>/<action:[\w\-]+>/<id:\d+>' => '<controller>/<action>',
'<controller:\w+>/<action:[\w\-]+>' => '<controller>/<action>',
```
In this app, there are so few endpoints (6?), spelling them out explicitly seems fine.